resources/views/nav.blade.php

```diff
@@ -13,6 +13,18 @@
                     </span>
                 </div>
             </form>
-        </div>
+            <form class="navbar-form navbar-right" role="search" method="get"
```

Comment:
Yeah, this looks horrible in mobile. Insomniac's suggestion in https://github.com/Kender2/swc-squadranking/issues/12#issuecomment-269651029 might be a better approach.